src/helpers/promise.js

```diff
@@ -1,4 +1,5 @@
 /* eslint no-proto:0 */
+/* eslint no-use-before-define:0 */
```

Comment:
I have added that because ```PromiseExt``` doesn't exist for eslint. Is certainly required by ```RSVP``` but is not goodly initialized.